main.py

```diff
@@ -516,6 +555,25 @@ async def get_master_dataset(key: str) -> Union[int, float, List]:
     return array
 
 
+@app.get("/dataset/master/list/")
+async def list_dataset() -> Tuple[str, ...]:
+    """Returns the list of datasets available in artiq master."""
+    return dataset_tracker.datasets()
+
+
+@app.get("/dataset/master/modification/")
+async def get_dataset_modification(
+    key: str, timestamp: float
+) -> Tuple[float, Tuple[dset.Modification, ...]]:
+    """Returns the dataset modifications since the given timestamp.
+    
+    Args:
+        key: The key of the target dataset.
+        timestamp: The timestamp of the last update. See dataset.DatasetTracker.since().
```

Comment:
It looks better to move "See dataset..." above "Args:" since it is an overall explanation of this function.